front/src/components/common/Button.tsx

```diff
@@ -1,41 +1,28 @@
 import styled from '@emotion/styled';
-import { Flex } from './Flex';
 import { KeyOfPalette, theme } from '@styles/theme';
 
-interface ButtonProps {
-  width: number;
-  type: 'large' | 'medium' | 'small';
-  isIcon?: boolean;
-  text: string;
+export const Button = styled.button<{
+  width?: number;
+  heightType?: string;
```

Comment:
```suggestion
  heightType?: 'large' | 'medium' | 'small';
```
와 같이 디테일하게 써주면 나중에 heightType props으로 넘겨줄 때 목록이 뜨니까 더 좋을 것 같아요!

Reply:
수정 완료했습니다